### ISSUE TITLE
Room Detection NOT working properly after Meteor Hit

### ISSUE THREAD

**Issue by Conchipedia** [StrangeLoopGames/EcoIssues#9501]
Version: 0.7.7.2 beta

why would it happen when Meteor Hits the World and makes Massive holes in a building it is still a room?

we just had the Meteor Hit and the Following 4 snapshots shows whats left of my main room (taken as to sort of overlap to show room and holes)
![snap2932](https://user-images.githubusercontent.com/29383891/45933451-fca41480-bf84-11e8-9dc0-21aa7cde7fed.png)
![snap2933](https://user-images.githubusercontent.com/29383891/45933452-fca41480-bf84-11e8-935a-e7ebdc1c6759.png)
![snap2934](https://user-images.githubusercontent.com/29383891/45933453-fca41480-bf84-11e8-9e4e-81e634eb9fff.png)
![snap2935](https://user-images.githubusercontent.com/29383891/45933454-fd3cab00-bf84-11e8-81e2-fd5e76e0a68b.png)

**Comment by Conchipedia** [comment]
it took a while after Meteor hit and now the room is broken as it should be
![snap2952](https://user-images.githubusercontent.com/29383891/46023773-4a339500-c0dd-11e8-9dc0-e27f97688249.png)
not sure when this happened as didn't log in for over a day

**Comment by johnkslg** [comment]
Meter hit is not causing rooms to reevaluate.  It should do this after craters are spawned.

**Comment by johnkslg** [comment]
Can test with /spawnbuilding and /spawncrater.  
When the crater is spawned, it changes the blocks in batch, thus bypassing the usual code that detects when a room is changed.  Youll have to manually trigger it after the crater is formed (in an efficient way)

**Comment by sidjuego** [comment]
@johnkslg 
We rechecked this issues in different scenarios, & We found this issue is fixed & working fine, kindly confirm the status.

**Comment by sidjuego** [comment]
I have debugged this bug in the following scenarios.
1: I tried spawning all tier buildings and spawned crates in the same place, Buildings gets destroyed immediately as intended.
2: I created different tier buildings and wrote the command for meteor impacts. Meteor impacts happen at random positions and it even destroyed half my building where the meteor hit; works as intended.
Hence, the issue seems to be fixed as it works fine.
